src/controller/AutoCommissioner.cpp

```diff
@@ -24,12 +24,14 @@
 #include <controller/CHIPDeviceController.h>
 #include <credentials/CHIPCert.h>
 #include <lib/support/SafeInt.h>
+#include <tracing/metric_event.h>
```

Comment:
Why the header and namespace change in this file?  I don't see any metrics being recorded in this file....